plugins/inputs/win_perf_counters/README.md

```diff
@@ -92,6 +104,14 @@ This must follow before other plugin configurations,
 beneath the main win_perf_counters entry, `[[inputs.win_perf_counters]]`.
 
 Following this are 3 required key/value pairs and three optional parameters and their usage.
+#### Computers
+*Optional*
+
+Host names or IPs of computers to gather counters from. User, under which Telegraf runs, must be already authenticated to the remote computer(s). E.g. via Windows sharing `net use \\SQL-SERVER-01`.
+Skip, if gather only from localhost.
+Use either empty string `""`, localhost `"localhost"` or real computer name to gather counters also from localhost among other computers.
+
+Example:  `Computers = ["localhost", "SQL-SERVER-01", "SQL-SERVER-02", "SQL-SERVER-03"]`
```

Comment:
Change case of `Computers` to lower

Reply:
It is for consistency with other params, which are also capitalized.